src/backend/src/utils/google-integration.utils.ts

```diff
@@ -88,6 +88,14 @@ export const uploadFile = async (fileObject: Express.Multer.File) => {
   const bufferStream = new stream.PassThrough();
   bufferStream.end(fileObject.buffer);
 
+  if (fileObject.filename.length > 20) {
+    throw new HttpException(400, 'File name is too long');
+  }
+  /* The regex /^[\w\-\s]+$/ limits the file name to the set of alphanumeric characters (\w), hyphens (-), spaces (\s), and dots */
+  if (!/^[\w\-\s.]+$/.test(fileObject.filename)) {
+    throw new HttpException(400, 'File name contains invalid characters');
```

Comment:
this is pretty nit-picky, but this error message could be a little more descriptive. Maybe something more like 'File name should only contain letters and numbers'?